### PR TITLE
xrd commands are deprecated, use xrdfs instead

### PR DESCRIPTION
**Pull request by smuzaffar**
`xrd` command is deprecated and it is suggested to use `xrdfs`. Few workflows[a] in 11.2 and 11.1 IBs are failing due to this. This PR suggests to use `xrdfs` command instead

[a] wf: 1370.* , 25125.*
LHE input from article  18334
Note: this tool is DEPRECATED, use xrdfs instead.
Issue to load LHE files, please check and try again.

### CODE REVIEW

**Comment by smuzaffar** [comment]
test parameters:
- workflows = 1370.0,1370.17,1370.18,25215.0,25215.17,25215.18

File: GeneratorInterface/LHEInterface/scripts/cmsLHEtoEOSManager.py [hunk 1]
```diff
@@ -12,15 +12,15 @@
 
 defaultEOSRootPath = '/eos/cms/store/lhe'
 defaultEOSLoadPath = 'root://eoscms/'
-defaultEOSlistCommand = 'xrd eoscms dirlist '
-defaultEOSmkdirCommand = 'xrd eoscms mkdir '
-defaultEOSfeCommand = 'xrd eoscms existfile '
+defaultEOSlistCommand = 'xrdfs eoscms ls '
```
Comment (kpedro88) [inline GeneratorInterface/LHEInterface/scripts/cmsLHEtoEOSManager.py:15]: `eoscms` will not be recognized universally

the suggestion in #31161 was to use `root://eoscms.cern.ch`, which is a concrete address

**Comment by smuzaffar** [comment]
please test

File: GeneratorInterface/LHEInterface/scripts/cmsLHEtoEOSManager.py (hunk 1 above)
Comment (smuzaffar) [reply GeneratorInterface/LHEInterface/scripts/cmsLHEtoEOSManager.py]: done

**Comment by makortel** [comment]
Apparently IBs are not the only ones having problems
https://hypernews.cern.ch/HyperNews/CMS/get/physics-validation/3580.html
Maybe this PR needs to be backported to (some of) earlier release cycles as well.

**Comment by agrohsje** [comment]
+generators

**Comment by qliphy** [comment]
+1

**Pull request merged**